addons/web/static/src/views/fields/text/text_field.xml

```diff
@@ -1,28 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-
     <t t-name="web.TextField" owl="1">
-        <t t-if="props.readonly">
-            <span t-esc="props.value or ''" />
-        </t>
-        <t t-else="">
-            <textarea
-                class="o_input"
-                t-att-class="{'o_field_translate': props.isTranslatable}"
-                t-att-id="props.id"
-                t-att-placeholder="props.placeholder"
-                t-att-rows="rowCount"
-                t-on-input="onInput"
-                t-ref="textarea"
-            />
-            <t t-if="props.isTranslatable">
-                <TranslationButton
-                    fieldName="props.name"
-                    record="props.record"
-                    updateField="props.update"
+        <div class="o_field_text_wrapper">
```

Comment:
Why do you need this wrapper?

Reply:
I'm using it because in mailing_sms we add some text under the field, and that throws off the absolute positioning. So I add this div to be able to position relative only to the input field alone.

Reply:
You can achieve the correct positioning without this, adding extra layers of wrappers is unnecessary. The previous emoji picker didn't need this, so you may look at that for inspiration, or you can just have a wrapper around the emoji picker itself only. In fact, if you keep your current css and just put the wrapper around the emoji picker button, the result is exactly the same without any need to touch the code of the text field.

Reply:
The previous approach of using a negative margin didn't work too well with buttons (previously we used spans) so I'd rather explicitly use absolute positioning and be sure of the result.

I'll follow your second suggestion if that's fine :)
thanks for the feedback.